cmd/gtk/dialog_wallet_create_address.go

```diff
@@ -21,13 +20,13 @@ func createAddress(ww *widgetWallet) {
 	addressLabel := getEntryObj(builder, "id_entry_account_label")
 
 	addressTypeCombo := getComboBoxTextObj(builder, "id_combo_address_type")
-	addressTypeCombo.Append(wallet.AddressTypeBLSAccount, "Account")
+	addressTypeCombo.Append(wallet.AddressTypeEd25519Account, "ED25519 Account")
```

Comment:
We can define String function for Address Types

Reply:
Address types in wallet is not enum and don't have specific type for define String method.

We can use crypto.AddressType in future.